src/TwilioVideo.android.js

```diff
@@ -255,6 +261,10 @@ class CustomTwilioVideoView extends Component {
     this.runCommand(nativeEvents.toggleSoundSetup, [speaker])
   }
 
+  onLocalParticipantH246NotSupported () {
+    this.runCommand(nativeEvents.onLocalParticipantH246NotSupported, [])
```

Comment:
Oh, here you have the "Not" in the name. Isn't the name supposed to match with [that one](https://github.com/brycnguyen/react-native-twilio-video-webrtc-1/pull/1/files#diff-6eb27f32b0cac807a5a9026f033bae71b9a6b9390942ed16f17dd3313a833b46R151)?

Reply:
This is not needed.